alpha/nodejs-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.deploy.applicationProperties.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "nodejs.name" . }}
+  labels:
+    {{- include "nodejs.labels" . | nindent 4 }}
+data:
+  application.properties: |-
+    {{- tpl .Values.deploy.applicationProperties.properties . | nindent 4 }}
+{{- end }}
```

Comment:
In Quarkus, this ConfigMap was for adding an externalized application.properties file. This doesn't apply for NodeJS, so it should be removed